gobblin-modules/gobblin-crypto/src/main/java/org/apache/gobblin/crypto/GPGFileDecryptor.java

```diff
@@ -113,23 +117,38 @@ public InputStream decryptFile(InputStream inputStream, InputStream keyIn, Strin
       throw new IllegalArgumentException("secret key for message not found.");
     }
 
+    ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+    InputStream resultStream = null;
+
     try (InputStream clear = pbe.getDataStream(
         new JcePublicKeyDataDecryptorFactoryBuilder().setProvider(BouncyCastleProvider.PROVIDER_NAME).build(sKey))) {
 
       JcaPGPObjectFactory pgpFact = new JcaPGPObjectFactory(clear);
       Object pgpfObject = pgpFact.nextObject();
 
-      if (pgpfObject instanceof PGPCompressedData) {
-        PGPCompressedData cData = (PGPCompressedData) pgpfObject;
-        pgpFact = new JcaPGPObjectFactory(cData.getDataStream());
+      while (pgpfObject != null)
+      {
```

Comment:
Please reformat the code per our [coding guide](https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/)